### ISSUE TITLE
[A11y] Search Results - Grid - Landmarks

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1922]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

Some relatively minor landmark issues:

- The "results navigation" landmark isn't really a navigation landmark - navigation landmarks are meant for collections of links.  This would make a lot of sense around the pagination, but it's a bit weird when it wraps options that just change how the results are displayed.
- The search landmark around filters should have a label so it's clear it's not a site search, like "filter results" or something.  This is probably not a failure, though, just a suggestion.
- The pagination at the bottom really should be in a navigation landmark with a label of "pagination"

### Suggested Fix

Note that I'm not sure any of these are hard failures, and only one is even a soft failure.  Mostly this is best-practice territory.  I think.

The main concern is the sort/display options using a navigation landmark.  Change that to either a generic (but labeled) region, or just make it not a region at all.

Adding a label to the "Search within the results" landmark might be nice, but I'm not sure it matters much.  Once a user reaches that landmark they'll get the information they need to know what that search does.

Wrap pagination in a navigation landmark so it's easy to find, and label it "pagination".  This helps give context to the links, too (otherwise those might be failures due to lack of context).

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Check landmarks using Accessibility Insights or a similar tool

**Comment by CGillen** [comment]
Fixed by #2154
